mlonmcu/target/riscv/etiss.py

```diff
@@ -66,6 +66,8 @@ class EtissTarget(RISCVTarget):
         "rom_size": 0x800000,  # 8 MB
         "ram_start": 0x1000000 + 0x800000,
         "ram_size": 0x4000000,  # 64 MB
+        "flash_start": 0x42000000,
+        "flash_size": 0x800000,  # 8 MB
```

Comment:
Can we set this to None by default and only add a flash segment if overridden manually?